Help/Hackathon.rst

```diff
@@ -0,0 +1,206 @@
+totally unsure what to use, simply visit
+`this link <https://github.com/coala/bear-docs#all>`__
+and click on each bear under your language's header. In a matter of minutes
+you should decide which bear to use.
+
+
+Now you are in your project's directory, have coala installed and know
+what bears you want to run. Here we learn about the two core arguments that
+we use to run coala: ``--files`` and ``--bears``. They are probably intuitive,
+from what their names' say.
+
+An example of a command which runs ``SpaceConsistencyBear`` on all ``C`` files
+in the current directory is:
+
+::
+
+  coala --files=*.c --bears=SpaceConsistencyBear
+
+Great, now coala should have yielded some results (supposing your code wasn't
+perfect), and they should prompt you in the terminal and ask you for actions.
+If you want to fix your code, you can give it actions (such as applying the
+patches found), but for the purpose of this tutorial simply seeing coala run
+should suffice.
+
+Contributing to coala
+---------------------
+
+Now you should be here in case you have completed the first step (running
+coala) or you have previously used coala.
+
+coala uses Github and Gitlab to maintain its repositories. During this
+hackathon we will only enlist the 3 most important repositories because
+these have accessible issues.
+
+coala is split in three main repositories:
+ - coalib, which is the core of the coala with its libraries
+ - coala-bears, which holds each bear, grouped on languages
+ - documentation, which has the documentation (even this page)
+
+To resolve an issue, you will have to locally clone the corresponding
+repository and work on its code.
+
+Picking up an Issue
+~~~~~~~~~~~~~~~~~~~
+
+Now let's pick up an issue. We use difficulty labels to mark issues, which
+go from "low" to "high". Of course, for this hackathon, you will choose a
+"low". In order to do so, visit coala.io/low and take a few moments to look
+through the issues. You should choose one that has noone
+assigned (even though in the past someone worked on it, if nobody is assigned,
+feel free to take it). If it appears inactive for at least one month, you can
+consider it unassigned.
+
+.. note::
+
+  To begin working on an issue, leave a comment that you would like to work on
+  it. A maintainer will then assign you to the issue. If someone else already
+  commented before you, you can assume that they are already claimed it.
+
+If you've made it this far, you already have an issue assigned to you.
+Now you gotta work on the issue. In order to do so, you should check which
+repository the issue you want to work on belongs to, from the 3 repos listed
+above. Clone it using:
+
+::
+
+  git clone https://github.com/coala/coala-bears
+
+or the correct link corresponding to the repository. Now you'll need an online
+copy of the repo (which is called a fork) which can be done by going on the
+repository page and clicking on the ``Fork`` button on the top right of the
+page.
+
+Fixing the issue
```

Comment:
Hmm, I think you might want to point them to the developer docs somewhere around here(like the development setup page)